pint.json

```diff
@@ -0,0 +1,172 @@
+{
+    "preset": "per",
```

Comment:
Laravel pint with `personal` settings.
They are not set in concrete, they are just **a** standard, not **the** standard